daemon/main.rs

```diff
@@ -87,6 +88,10 @@ async fn run(config: CliArgs) -> Result<RunOutput> {
         rng_seed: None,
     };
 
+    if !Regex::new(r"^[0-9a-fA-F]{88}$").unwrap().is_match(&runtime_config.station_id) {
+        return Err(anyhow!("Invalid station_id format"));
+    }
+
```

Comment:
Let's move the validation to the top of the function, please. We already have some validation code on lines 36-43.

Reply:
Ah I meant to leave a comment for this.

My reasoning was that if we had it with the other validations, we are not testing our own default value in the BootstrapOptions constructor.

Not sure if that has any value, what do you think?

Reply:
Ah, that's an interesting point. 

I agree it's useful to double-check the default value, too. But I don't think that's happening right now, because the default value is not used by `zinniad`.

Let's move the assertion to the function `run_js_module`, that's the entry point used by both `zinnia` and `zinniad`.